lib/inc/sairedis.h

```diff
@@ -194,4 +194,15 @@ typedef enum _sai_redis_switch_attr_t
      */
     SAI_REDIS_SWITCH_ATTR_CONTEXT,
 
+    /**
+     * @brief Recording log filename.
+     *
+     * Default valus is sairedis.rec
+     *
+     * @type sai_s8_list_t
+     * @flags CREATE_AND_SET
+     * @default empty
```

Comment:
you could make this @default "sairedis.rec"

Reply:
address the commit in the latest commit